test/unit/typeahead_spec.js

```diff
@@ -40,6 +40,20 @@ describe('Typeahead', function() {
     this.dropdown = this.view.dropdown;
   });
 
+
+  describe('appendTo', function() {
```

Comment:
Did you try to add a unit test that checks that something is added to a particular node?

```js
var node = document.createElement('div');
document.querySelector('body').appendChild(node);

// instantiate with appendTo..
// test node now has children?..
```